examples/train_brats2018_new.py

```diff
@@ -38,18 +39,25 @@ def get_arguments():
     parser.add_argument('--dataset_name', type=str, default="brats2018")
     parser.add_argument('--dim', nargs="+", type=int, default=(64, 64, 64))
     parser.add_argument('--nEpochs', type=int, default=100)
-    parser.add_argument('--classes', type=int, default=5)
-    parser.add_argument('--samples_train', type=int, default=100)
-    parser.add_argument('--samples_val', type=int, default=100)
+    parser.add_argument('--classes', type=int, default=4)
```

Comment:
just writing things that I think. so maybe we can consider them in the future. the convention is to use both `--` and `-`. double for full names and single for letters. e.g `--dataset-name` and `-d`

Reply:
i was not aware of this! fair point!!!!!